### PR TITLE
Feature/form

### PR DESCRIPTION
**Pull request by tommasodovidio**
Updated feature/form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,83 +5,80 @@
...
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css" integrity="sha512-UJfAaOlIRtdR+0P6C3KUoTDAxVTuy3lnSXLyLKlHYJlcSU8Juge/mjeaxDNMlw9LgeIotgz5FP8eUQPhX1q10A==" crossorigin="anonymous" />
     ​<script src="https://code.jquery.com/jquery-3.5.1.min.js"   integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="   crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
     <title>Daily-Dashboard</title>
 </head>
 
-<body> ​
+<body> 
     <header>
-        <h2>Welcome back, Amy!</h2>
```
Comment (SterlynKong) [inline index.html:17]: Consider reading this to display name captured in Modal

File: index.html [hunk 2]
```diff
@@ -5,83 +5,80 @@
...
+            <input type="text" required title="Please enter your city" size="48" name="city" value=""></label></p>
+            <p><input type="submit" name="modal_form" value="Submit"></p>
+            </form>
+
+            </div> <!-- #modal_window -->
+            </div> <!-- #modal_wrapper -->
+
     </header>
     ​
-    <section class="weather wrapper">
```
Comment (SterlynKong) [inline index.html:22]: Amy's code may be referencing the class "weather-wrapper" and may break due to this change. Please take note, Amy.

File: index.html [hunk 3]
```diff
@@ -5,83 +5,80 @@
...
-            <!--/Daily forecast temperatures-->
-            <div class="forecast">
-                <div id="chart-container">Daily forecast chart</div>
-                <!--canvas element for where the chart will be rendered-->
-                <div class="chart" style="position: relative; height:50vh; width:50vw">
-                    <canvas id="myChart"></canvas>
-                </div>
-                <!--end of canvas element for chart-->
-            </div>
+            <!--attempt at collapsible drawer for "favourite" (recently searched) cities-->
```
Comment (SterlynKong) [inline index.html]: Removed duplicate collapsible draw.

**Review by SterlynKong — APPROVED** [review]
Some changed to be reconsidered but good for now.

**Review by leggeamy — APPROVED** [review]

**Pull request merged**